v2/brigadier/src/logger.ts

```diff
@@ -1,25 +1,53 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+
+/**
+ * Provides logging services for a script.
+ * 
+ * Access the logger through the global `logger` object.
+ */
```

Comment:
Another case where idk if it's worth mentioning that we polyfill this at runtime.

This implementation does `console.debug(...)` and `console.error(...)`, etc... and iirc, there's actually no difference between what each of those does, right? They all right to stdout/stderr and don't have any real concept of log severity and whether the message in question should be written out at all or not?

The polyfill that the worker uses relies upon a leveled logging library (winston, iirc) to get this right. We _don't rely on it here in Brigadier because that library drags too many transitive dependencies along with it.